discovery/discovery.go

```diff
@@ -148,12 +148,12 @@ func doDiscover(app App, pre string, insecure bool) (*Endpoints, error) {
 // DiscoverEndpoints will make HTTPS requests to find the ac-discovery meta
```

Comment:
Yeah, for now I think it should be opaque to users, so please combine them. If simple discovery fails with an error (as opposed to just not finding anything), it should fail immediately imho.